src/status_im/contexts/wallet/add_address_to_watch/view.cljs

```diff
@@ -11,19 +11,16 @@
     [utils.i18n :as i18n]
     [utils.re-frame :as rf]))
 
-(defn validate-message
-  [addresses]
-  (fn [s]
-    (cond
-      (or (= s nil) (= s ""))             nil
-      (contains? addresses s)             (i18n/label :t/address-already-in-use)
-      (not (or (validation/eth-address? s)
-               (validation/ens-name? s))) (i18n/label :t/invalid-address)
-      :else                               nil)))
+(defn- validate-address
+  [known-addresses s]
+  (cond
+    (or (nil? s) (= s ""))              nil
```

Comment:
nit: (= s "") --> (string/blank? s)

Reply:
thanks @ibrkhalil !
Actually, I already tried it, but they are different and I didn't want to change the logic.

`blank` will say `true` when it's `"                "` or `"\n \n"` too, I thought it was cheaper to just comapre it with `=` nad `""`